x-pack/plugins/security_solution/server/lib/detection_engine/rule_management/logic/actions/duplicate_rule.test.ts

```diff
@@ -111,10 +111,10 @@ describe('duplicateRule', () => {
     });
   });
 
-  it('appends [Duplicate] to the name', () => {
+  it('appends [Duplicate] to the name', async () => {
     const rule = createTestRule();
     rule.name = 'PowerShell Keylogging Script';
-    const result = duplicateRule(rule);
+    const result = await duplicateRule(rule, false, undefined);
```

Comment:
Let's make the `duplicateRule` function accept its parameters as an object. Like:

```suggestion
    const result = await duplicateRule({ rule, shouldDuplicateExceptions: false, exceptionsClient: undefined });
```

Because when passed primitives, it makes it pretty hard to read and understand the meaning.